pep-0687.rst

```diff
@@ -67,6 +67,9 @@ All stdlib extension modules will be *isolated*. That is:
 Conversion to heap types
 ------------------------
 
+Static types that do not need module state access, and have no other reason to
+be converted, should stay static.
+
```

Comment:
@gvanrossum: Does this change, together with the sentence below, add enough emphasis for you?

Reply:
Another option would be to put the above sentence in a `note`. That would definitely highlight it.